core/modules/qmeta/QMeta.cc

```diff
@@ -15,6 +15,8 @@
 #include "util/ConfigStore.h"
 #include "util/ConfigStoreError.h"
 
+using namespace std;
+
```

Comment:
I'd probably revert most changes to `qmeta`, I think your ticket is only supposed to touch `css`. Some of these look like style-only changes which don't need to be on this ticket.